src/main/java/Controller/DukeController.java

```diff
@@ -0,0 +1,186 @@
+package Controller;
+import Tasks.*;
+
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+public class DukeController {
+    private Parser parse = new Parser();
+    public DukeController() {};
+
+    public void printTask(ArrayList<Task> tasks, int count) {
+        System.out.println("--------------------------------------------");
+        System.out.println("Got it. I've added this task: ");
+        System.out.println(tasks.get(count).getPrintedLine());
+        System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+        System.out.println("--------------------------------------------");
+    }
+
+    public void printList(ArrayList<Task> tasks) {
+        int count = 1;
+        System.out.println("--------------------------------------------");
+        System.out.println("Here are the tasks in your list:");
+        for (Task task: tasks) {
+            System.out.println(count+ ". " + task.getPrintedLine());
+            count++;
+        }
+        System.out.println("--------------------------------------------");
+    }
+
+    public void printDone(ArrayList<Task> tasks, String in) {
```

Comment:
Could try to use more descriptive names for function parameters (i.e. String in)